.happy/terraform/modules/deletion/variables.tf

```diff
@@ -8,6 +8,12 @@ variable task_role_arn {
   description = "ARN for the role assumed by tasks"
 }
 
+variable execution_role {
+  type        = string
+  description = "Execution role to use for fargate tasks - required for fargate services!"
+  default     = ""
```

Comment:
Should we make this a required parameter? Otherwise, should we make an assert statement that if they choose to use fargate this variable needs to be set? I feel like an empty string might lead to unexpected bugs at apply time.